src/tribler-core/tribler_core/upgrade/tests/test_config_upgrade_to_75.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+
+from tribler_core.config.tribler_config import TriblerConfig
+from tribler_core.tests.tools.base_test import TriblerCoreTest
+from tribler_core.tests.tools.common import TESTS_DATA_DIR
+from tribler_core.upgrade.config_converter import convert_config_to_tribler76
+
+
+class TestConfigUpgradeto75(TriblerCoreTest):
```

Comment:
Should be UpgradeTo**76** and in a separate file, like `test_config_upgrade_to76.py`

Reply:
Oops, copy-paste error!